platforms/ide/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/ClientForwardingBuildOperationListener.java

```diff
@@ -70,11 +74,20 @@ protected DefaultOperationDescriptor toBuildOperationDescriptor(BuildOperationDe
     }
 
     static AbstractOperationResult toOperationResult(OperationFinishEvent result) {
+        return toOperationResult(result, null);
+    }
+
+    static AbstractOperationResult toOperationResult(OperationFinishEvent result, Map<Throwable, Collection<Problem>> problemsForThrowables) {
```

Comment:
❗Needs import, add this manually
```suggestion
    static AbstractOperationResult toOperationResult(OperationFinishEvent result, @Nullable Map<Throwable, Collection<Problem>> problemsForThrowables) {
```

Reply:
Fixed